src/YoutubeAtom.tsx

```diff
@@ -311,12 +295,15 @@ export const YoutubeAtom = ({
 
             {(overlayImage || posterImage) && (
                 <div
-                    onClick={() => player?.playVideo()}
+                    onClick={() => {
+                        setHasUserLaunchedPlay(true);
```

Comment:
Does this need to be a state value? Perhaps we can just have it as a standard variable inside the useEffect, along with the other progress trackers?